routes/question-route.js

```diff
@@ -0,0 +1,12 @@
+var express = require ("express");
+var questionRouter  = express.Router();
+var Question = require('../models/question.js');
+
+
+questionRouter.get('/',(req,res)=>{
+    question.findAll().then((questions)=>{
+        res.send(questions)
+    })
+})
+
+module.exports= questionRouter;
```

Comment:
Nice organization by splitting routes into separte files. Where are the POST routes though?